test/test-fork.c

```diff
@@ -346,6 +346,48 @@ TEST_IMPL(fork_signal_to_child_closed) {
   return 0;
 }
 
+static void fork_signal_cb(uv_signal_t* h, int s) {
+  fork_signal_cb_called = s;
+}
+static void empty_close_cb(uv_handle_t* h){}
+
+TEST_IMPL(fork_close_signal_in_child) {
+  uv_loop_t loop;
+  uv_signal_t signal_handle;
+  pid_t child_pid;
+
+  uv_loop_init(&loop);
+  uv_signal_init(&loop, &signal_handle);
+  uv_signal_start(&signal_handle, &fork_signal_cb, SIGHUP);
+
+  fork_signal_cb_called = 0;
+
+  kill(getpid(), SIGHUP);
```

Comment:
```suggestion
  ASSERT_EQ(0, kill(getpid(), SIGHUP));
```
Not that it can really fail but better safe than sorry.